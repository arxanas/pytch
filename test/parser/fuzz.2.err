INVALID_TOKEN[1000] in test/parser/fuzz.2.pytch, line 1, character 18:
Error: Invalid token '!'.
   +----------------------------------------------+
   | test/parser/fuzz.2.pytch                     |
 1 | let foo = print(1!                           |
   |                  ^ Error: Invalid token '!'. |
   +----------------------------------------------+
EXPECTED_COMMA[1017] in test/parser/fuzz.2.pytch, line 1, character 18:
Error: I was expecting a ',' after the previous argument.
   +--------------------------------------------------------------------+
   | test/parser/fuzz.2.pytch                                           |
 1 | let foo = print(1!                                                 |
   |                  ^ Error: I was expecting a ',' after the previous |
   | argument.                                                          |
   +--------------------------------------------------------------------+
