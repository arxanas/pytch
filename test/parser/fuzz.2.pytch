let foo = print(1!
