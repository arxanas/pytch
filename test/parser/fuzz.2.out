SyntaxTree
    LetExpr
        Token 'let'
        VariablePattern
            Leading ' '
            Token 'foo'
        Leading ' '
        Token '='
        FunctionCallExpr
            IdentifierExpr
                Leading ' '
                Token 'print'
            ArgumentList
                Token '('
                Argument
                    IntLiteralExpr
                        Token '1'
                    <missing>
                <missing>
        Leading '!\n'
        Token DUMMY_IN_FOR_LET ''
        <missing>
    Token EOF ''
